cocos/scripting/js-bindings/jswrapper/config.hpp

```diff
@@ -49,14 +50,14 @@
             #define SCRIPT_ENGINE_TYPE           SCRIPT_ENGINE_JSC
         #endif
     #endif
-
+#elif defined(OPENHARMONY)
```

Comment:
Why not use `CC_TARGET_PLATFORM == CC_PLATFORM_OPENHARMONY` ?

Reply:
Because the use of CC_PLATFORM_OPENHARMONY needs to include other .h files, which causes redundant include.

Reply:
Because the use of CC_PLATFORM_OPENHARMONY needs to include other .h files, which causes redundant include.